portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Personal Portfolio</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Serif&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link href="style.css" rel="stylesheet" type="text/css" />
+    <script src="script.js"></script>
+
+  </head>
+  <body>
+    <div  class="mainIntro">
+      <div class="navbar">
+        <img src="images/logo.png" class="logo" alt="myLogo">
+        <nav>
+          <ul id="menu">
+            <li><a href="index.html" id="home">Home</a></li>
+            <li><a href="about.html" id="about">About</a></li>
+            <li><a href="projects.html" id="project">Projects</a></li>
+            <li><a href="contact.html" id="contact">Contact</a></li>
+          </ul>
+        </nav>
+        <img src="images/menuLogo.png" class="menuLogo" onclick="toggleMenu()" alt="bars">
+      </div>
+    </div>
```

Comment:
This header looks like it is reused across multiple pages. You don't have to make any changes now, but if you're interested, there are ways to make a reusable component so that any changes you want to make to the header are done in one place.